tests/input_test.cpp

```diff
@@ -0,0 +1,42 @@
+//
+// Created by grzegorz on 17.12.18.
+//
+
+#include "pch.h"
+#include "gtest/gtest.h"
+
+#include "input_manager.h"
+
+
+TEST(InputManager, initialization){
+    using namespace Bald;
+    using namespace Input;
+
+    EXPECT_FALSE(InputManager::isKeyPressed(GLFW_KEY_SPACE));
+    EXPECT_FALSE(InputManager::isKeyTyped(GLFW_KEY_1));
+    EXPECT_FALSE(InputManager::isMouseButtonPressed(GLFW_MOUSE_BUTTON_1));
+    EXPECT_FALSE(InputManager::isMouseButtonTyped(GLFW_MOUSE_BUTTON_2));
+
+}
+
+
+TEST(InputManager, callback){
+    using namespace Bald;
+    using namespace Input;
+
+
+    glfwInit();
+
+    GLFWwindow* window = glfwCreateWindow(800, 600, "LearnOpenGL", NULL, NULL);
+    glfwMakeContextCurrent(window);
+
+    EXPECT_TRUE(glfwSetKeyCallback(window, key_callback) == NULL);
```

Comment:
Why not EXPECT_THAT(glfwSet..., NULL) ?
https://github.com/google/googletest/blob/master/googlemock/docs/CheatSheet.md#matchers